src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -206,30 +237,53 @@ let getPrevAnnotationTable (context:RequestContext) =
 // I subtract from the index of the current worksheet the indices of the other found worksheets with annotationTable.
 // I sort by the resulting lowest number (since the worksheet is then closest to the active one), I find the output column in the particular
 // annotationTable and use the values it contains for the new annotationTable in the active worksheet.
-let getPrevTableOutput (context:RequestContext) =
+
+let getPrevTable (context:RequestContext) =
     promise {
-        let! prevTableName = getPrevAnnotationTable context
+        let! prevTableName = getPrevAnnotationTableName context
 
         if prevTableName.IsSome then
-            // Ref. 2
-            let! buildingBlocks = BuildingBlockFunctions.getBuildingBlocks context prevTableName.Value
+        
+            let! result = ArcTable.arcTableFromExcelTableName(prevTableName.Value, context)
+            return result
+
+        else
+
+            let! result = ArcTable.arcTableFromExcelTableName("", context)
```

Comment:
what does this logic do? Why do we get an arctable from ""-name. Shouldn't this be tryGet? With Some/None?